cdap-app-fabric/src/main/java/io/cdap/cdap/app/preview/PreviewRunnerModule.java

```diff
@@ -156,6 +160,12 @@ protected void configure() {
 
     bind(PipelineFactory.class).to(SynchronousPipelineFactory.class);
 
+    install(
+      new FactoryModuleBuilder()
+        .implement(Configurator.class, InMemoryConfigurator.class)
+        .build(Key.get(ConfiguratorFactory.class, Names.named("local")))
```

Comment:
If preview always use this one, we shouldn't need an annotation.

Reply:
done